client/components/themes-list/index.jsx

```diff
@@ -78,17 +78,24 @@ export const ThemesList = ( props ) => {
 			goes_to_assembler_step: shouldGoToAssemblerStep,
 		} );
 
-		const basePathname = isLoggedIn ? '/setup' : '/start';
-		const params = new URLSearchParams( {
-			ref: 'calypshowcase',
-			theme: BLANK_CANVAS_DESIGN.slug,
-		} );
+		let destinationUrl;
+		if ( ! isLoggedIn && shouldGoToAssemblerStep ) {
```

Comment:
Can we add a comment to this condition? It seems to mean that LoTS mobile view seems to directly go to `siteEditorUrl`, but I don't think that's intended since the user is logged out. So the `siteEditorUrl` will be `nullsite-editor.php`

Reply:
Oops...the condition is wrong. It should be `! isLoggedIn || shouldGoToAssemblerStep` 😓